trento/xml/article_sap_trento.xml

```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
+      </listitem>
+          </listitem>
+          <listitem>
+        <para>For changes in the Trento Server new checks engine component (wanda), visit <link xlink:href="https://github.com/trento-project/wanda/releases"/></para>
+      </listitem>
+    </listitem>
+          <listitem>
+        <para>For changes in the Trento Agent, visit <link xlink:href="https://github.com/trento-project/agent/releases"/></para>
+      </listitem>
+    </itemizedlist>
     <!-- HINT: The list should be sorted by versions, having the most recent version at the
                first item.
     -->
     <variablelist>
       <varlistentry>
         <term>Version 2.1.0</term>
+       <para> Consisting of Helm chart 2.1.0, web 2.1.0, checks engine (wanda) 1.1.0 and agent 2.1.0. Released on 2023/08/02.</para>
+        <listitem>
+          <para>ASCS/ERS cluster discovery, from single-sid, two-node scenarios to multi-sid, multi-node setups. The discovery covers both versions of the enqueue server,
+          ENSA1 and ENSA2, and both scenarios with resource managed instance filesystems and simple mount setups.</para>
+        </listitem>
+               <listitem>
+          <para>Host clean-up capabilities, allowing users to get rid of hosts that are no longer part of their SAP environment.</para>
+        </listitem>
+       <listitem>
+          <para>New checks results view that leverages the potential of the new checks engine (wanda) and provides the user with insightful information about the 
+           check execution.</para>
+        </listitem>
+      </varlistentry>
+           <varlistentry>
+        <term>Version 2.0.0</term>
+       <para> Consisting of Helm chart 2.0.0, web 2.0.0, cchecks engine (wanda) 1.0.0 and agent 2.0.0. Released on 2023/04/26.</para>
         <listitem>
-          <para></para>
+          <para>A brand-new safer, faster, ligher SSH-less configuration checks engine (wanda) which not only opens the door to configuration checks for other HA scenarios (ASCS/ER, HANA scaleup
+           cost optimized, HANA scale-out) and other targes in the environment (hosts, HANA databases, NetWeaver instances), but also will allow customization of existing checks and
+          addtion of custom checks.</para>
+        </listitem>
+               <listitem>
+          <para>Addtion of VMware to the list of known platforms.</para>
+        </listitem>
+       <listitem>
+          <para>Versioned external APIs for both the web and the checks engine (wanda) components. The full list of available APIs can be found at 
+          https://www.trento-project.io/web/swaggerui/ and https://www.trento-project.io/wanda/swaggerui/ respectiverly.</para>
+        </listitem>
+      </varlistentry>
+     <varlistentry>
+        <term>Version 1.2.0</term>
+       <para> Consisting of Helm chart 1.2.0, web 1.2.0, runner 1.1.0 and agent 1.1.0. Released on 2022/11/04.</para>
```

Comment:
I think the version _and_ the release date are something the reader is interested in. As such, it should be more prominent and not hidden in a sentence:

```suggestion
       <term>Version 1.2.0, released on 2022/11/04</term>
       <para> Consisting of Helm chart 1.2.0, web 1.2.0, runner 1.1.0 and agent 1.1.0. </para>
```